packages/@okta/vuepress-site/docs/guides/implement-oauth-for-okta-serviceapp/create-publicprivate-keypair/index.md

```diff
@@ -0,0 +1,37 @@
+---
+title: Create a public/private key pair
+---
+
+The `private_key_jwt client` authentication method is the only supported method for OAuth service apps that want to get access tokens with Okta scopes.
```

Comment:
Shouldn't this be `private_key_jwt` client authentication instead of `private_key_jwt client` authentication?

Reply:
yep, thank you.